examples/pytorch/stgcn_wave/main.py

```diff
@@ -0,0 +1,114 @@
+import dgl
+import networkx as nx
+import random
+import torch
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+from load_data import *
+from utils import *
+from model import *
+from sensors2graph import *
+import torch.nn as nn
+import argparse
+
+parser = argparse.ArgumentParser(description='STGCN_WAVE')
+parser.add_argument('--lr', default=0.001, type=float, help='learning rate')
+parser.add_argument('--disablecuda', action='store_true', help='Disable CUDA')
+parser.add_argument('--batch_size', type=int, default=50, help='batch size for training and validation (default: 50)')
+parser.add_argument('--epochs', type=int, default=50, help='epochsfor training  (default: 50)')
+args = parser.parse_args()
+
+device = torch.device("cuda") if torch.cuda.is_available() and not args.disablecuda else torch.device("cpu")
+
+with open('./data/sensor_graph/graph_sensor_ids.txt') as f:
+    sensor_ids = f.read().strip().split(',')
+distance_df = pd.read_csv('./data/sensor_graph/distances_la_2012.csv', dtype={'from': 'str', 'to': 'str'})
+
+_, sensor_id_to_ind, adj_mx = get_adjacency_matrix(distance_df, sensor_ids)
+ng = nx.from_numpy_matrix(adj_mx)
+G = dgl.DGLGraph(ng)

```

Comment:
can we avoid using networkx here? we can pass a scipy sparse matrix to DGL directly.